core/src/main/java/hudson/TcpSlaveAgentListener.java

```diff
@@ -201,8 +201,6 @@ public ConnectionHandler(Socket s) {
         @Override
         public void run() {
             try {
-                LOGGER.log(Level.INFO, "Accepted connection #{0} from {1}", new Object[]{id,s.getRemoteSocketAddress()});
```

Comment:
Better to lower the log lever to FINE. This can be useful when diagnosing some issues connecting.